pkg/param/param.go

```diff
@@ -294,13 +297,28 @@ func fetchRepositoryServer(ctx context.Context, cli kubernetes.Interface, crCli
 		return nil, errors.Wrap(err, "Error Fetching Repository Server Service")
 	}
 	repositoryServerAddress := fmt.Sprintf("https://%s.%s.svc.cluster.local:%d", repositoryServerService.Name, repositoryServerService.Namespace, repositoryServerService.Spec.Ports[0].Port)
```

Comment:
Lets add the domain `svc.cluster.local` as a constant

Reply:
Done !